h2o-r/h2o-package/R/models.R

```diff
@@ -2508,15 +2512,15 @@ h2o.giniCoef <- function(object, train=FALSE, valid=FALSE, xval=FALSE) {
 #' }
 #' @export
 h2o.coef_with_p_values <- function(object) {
-  if (is(object, "H2OModel") && object@algorithm %in% c("glm")) {
-    if (object@parameters$compute_p_values) {
-      object@model$coefficients_table
+    if (is(object, "H2OModel") && object@algorithm %in% c("glm")) {
+        if (object@parameters$compute_p_values) {
+            object@model$coefficients_table
+        } else {
+            stop("p-values, z-values and std_error are not found in model.  Make sure to set compute_p_values=TRUE.")
+        }
     } else {
-      stop("p-values, z-values and std_error are not found in model.  Make sure to set compute_p_values=TRUE.")
+        stop("p-values, z-values and std_error are only found in GLM.")
```

Comment:
Please add

Make sure to set compute_p_values=TRUE.

Reply:
The PR is still a draft and not rebased on the actual master. So after that, this will be resolved.